packages/bpk-component-card/src/BpkDividedCard.module.scss

```diff
@@ -59,4 +59,14 @@ $fixed-secondary-width: 216;
       }
     }
   }
+
+  &--depressed {
+    box-shadow: none;
+
+    @include bpk-hover {
```

Comment:
it seems like on exiting the hover state the shadow re-appears for a brief moment

Reply:
Nice catch, it due to shadow in pseudo element, have depressed it.